libs/core/message-toast/message-toast.service.spec.ts

```diff
@@ -1,6 +1,6 @@
-/* eslint-disable no-unused-vars */
-/* eslint-disable @typescript-eslint/no-unused-vars */
-/* eslint-disable @angular-eslint/component-class-suffix */
+ 
```

Comment:
@droshev Empty lines - i'm surprised these don't get removed automatically by lint

Reply:
this was done by the tooling with lint --fix